docs/data/joy/components/table/table.md

```diff
@@ -0,0 +1,272 @@
+:::
+
+### Sizes
+
+The component comes in three sizes: `sm`, `md` (default), and `lg`.
+
+{{"demo": "TableSizes.js"}}
+
+:::info
+To learn how to add custom sizes to the component, check out [Themed components—Extend sizes](/joy-ui/customization/themed-components/#extend-sizes).
+:::
+
+### Stripe
+
+{{"demo": "TableStripe.js"}}
+
+:::success
+
+The prop also supports complex argument of [`:nth-child()`](https://developer.mozilla.org/en-US/docs/Web/CSS/:nth-child#syntax) CSS selector.
+
+For example, you can use `3n` as a value to create stripes on row number three, six, nine, and so on:
+
+```js
+<Table stripe="3n">
+```
+
+:::
+
+### Hover
+
+To highlight a row of the table body when hover on it, set `hoverAxis` prop to true.
+
+{{"demo": "TableHover.js"}}
+
+### Border
+
+Use `borderAxis` prop to control the border appearance.
+
+{{"demo": "TableBorder.js"}}
+
+<details>
+<summary><b>Adding custom values</b></summary>
+
+Customize the table theme based on `borderAxis` prop using [`extendTheme()`](/joy-ui/customization/themed-components/#change-styles-based-on-props) function.
+
+```js
+import { CssVarsProvider, extendTheme } from '@mui/joy/styles';
+
+const theme = extendTheme({
+  components: {
+    JoyTable: {
+      styleOverrides: {
+        root: ({ ownerState }) => ({
+          ...(ownerState.borderAxis === 'header' && {
+            '& thead th:not([colspan])': {
+              border: '2px solid var(--TableCell-borderColor)',
+            },
+          }),
+        });
+      }
+    }
+  }
+})
+
+<CssVarsProvider theme={theme}>…</CssVarsProvider>
+```
+
+For **TypeScript**, you have to add the new values via module augmentation:
+
+```ts
+// this could be any file that's included in your tsconfig.json
+declare module '@mui/joy/Table' {
+  interface TablePropsBorderAxisOverrides {
+    header: true;
+  }
+}
+```
+
+</details>
+
+### Sticky header
+
+Set `stickyHeader` prop to true. The table must be a child of a fixed-height element with overflow `auto` (or `scroll`).
+
+Using [Sheet](/joy-ui/react-sheet/) component is recommended for wrapping a table. See [usage with Sheet](#usage-with-sheet) to learn more.
+
+{{"demo": "TableStickyHeader.js"}}
+
+### Caption
+
+Place `<caption>` as the first child of the table.
+
+{{"demo": "TableCaption.js"}}
+
+:::success
+To display caption at the bottom of the table, set [caption side](https://developer.mozilla.org/en-US/docs/Web/CSS/caption-side) to `bottom`:
+
+```js
+<Table sx={{ captionSide: 'bottom' }}>
+```
+
+:::
+
+### Footer
+
+Use [`<tfoot>`](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/tfoot) to show the table footer.
+
+{{"demo": "TableFooter.js"}}
+
+### Row head
+
+Set `scope="row"` to `th` elements inside `tbody` to apply the same style as column headers.
+
+{{"demo": "TableRowHead.js"}}
+
+### Row and column span
+
+Use `rowSpan` and `columnSpan` to expand a cell to multiple rows or columns.
+
+{{"demo": "TableRowColumnSpan.js"}}
+
+:::info
```

Comment:
Maybe `:::warning` is more appropriate, since it highlights a potential problem?

```suggestion
:::warning
```